scripts/install.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+set -u
+
+BYTEBASE_VERSION="1.1.1"
```

Comment:
reference scripts/VERSION instead?

Reply:
The user will not download the Bytebase repo, only uses`/bin/bash -c "$(curl -fsSL https://xxxxxxx)`. So reference may not be a good option.

Reply:
Done.